test_create_geometry.py

```diff
@@ -0,0 +1,31 @@
+import gmsh
+import sys
+import pytest
+from create_geometry_3D import make_geometry_2D
+from create_geometry_3D import extract_mesh_data
+
+
+def test_make_geometry():
+    mesh_type = "triangular"
+    lc = 2
+    points = [(0, 0, 0), (1, 0, 0), (1, 3, 0), (0, 3, 0), (-1, 1.5, 0)]
+    point_pairs = []
+    for i in range(len(points) - 1):
+        # select two consecutive points and store them in a list
+        point_pair = [i + 1, i + 2]
+        point_pairs.append(point_pair)
+
+    point_pairs.append([len(points), 1])
+    gmsh.initialize()
+
+    make_geometry_2D(points, point_pairs, lc)
+    gmsh.model.geo.synchronize()
+    gmsh.model.mesh.generate(2)
```

Comment:
please avoid input of raw numbers, create a new variable instead e.g. ndim = 2; gmsh.model.mesh.generate(ndim); or if the "2" is not the number of dimensions, please choose an appropriate name